ethereum_history_api/oracles/src/noir/oracles/oracles.ts

```diff
@@ -34,4 +34,8 @@ export const defaultOraclesMap: OracleMap = {
   get_transaction: getTransactionOracle
 };
 
-export const defaultOracles = createOracles(MultiChainClient.from_env())(defaultOraclesMap);
```

Comment:
`defaultOracles` was evaluating and throwing errors from `from_env()`.
Transformed it into function, so our SDK library can import stuff. 

Besides that `defaultOracles` is not used anywhere. Perhaps we may just remove it?